picard/tagger.py

```diff
@@ -1034,6 +1044,8 @@ def process_picard_args():
                         help="location of the configuration file")
     parser.add_argument("-d", "--debug", action='store_true',
                         help="enable debug-level logging")
+    parser.add_argument("-f", "--force", action='store_true',
```

Comment:
I like your idea of `--new-instance` more than `--new-window` because IMHO it more closely descibes the situation.  From my understanding, this new instance is stand-alone and does not listen for further commands.  Should this be clarified in the option name, something like `--stand-alone-instance` ?

Reply:
> --stand-alone-instance

I like this one.

Reply:
Me too, thanks

Reply:
Any ideas for `-f` replacement or do we keep only the long version?

Reply:
When I thought about it for a bit longer, you're right. The short option would make it just worse.

Reply:
Just wanted to note that I like `-s`  and `--stand-alone-instance`